Inspicio/wwwroot/js/quickCreate.js

```diff
@@ -91,10 +91,13 @@
             __RequestVerificationToken: $('[name= "__RequestVerificationToken"]').val(),
             CreatePageModel: CreatePageModel
         }
+        var controllers = ["Images", "Account", "Images", "Manage"];
+        var path = location.pathname.split("/");
+        alert("http://" + location.host + path[1] + "/Images/Create");
         $.ajax(
             {
                 type: "POST", //HTTP POST Method  
-                url: "http://"+location.host+"/Images/Create", // Controller  
+                url: path[1]+"/Images/Create", // Controller  
```

Comment:
This seems like it won't work when it's running at the root (e.g. the URL is http://localhost/Images/View), but I think it's worth going with this so we can get some feedback from watching people try it.